controllers/account/ec2.go

```diff
@@ -774,3 +792,62 @@ func cleanRegion(client awsclient.Client, logger logr.Logger, accountName string
 	}
 	return cleaned, nil
 }
+
+// Get the free instance type for the client's region
+func RetrieveAvailableMicroInstanceType(logger logr.Logger, awsClient awsclient.Client) (string, error) {
+	// FIXME: For unknown reasons attempting to use the free-tier-eligible
+	// filter from go returns *nothing*, but works fine from the CLI.
+	// HTTP-requests looks the same using both options.
+	availableTypes, err := awsClient.DescribeInstanceTypes(&ec2.DescribeInstanceTypesInput{
+		InstanceTypes: []*string{aws.String("t3.micro")},
+	})
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case "InvalidInstanceType":
+				logger.Info("Did not find t3.micro - falling back to t2.micro")
+				availableTypes, err := awsClient.DescribeInstanceTypes(&ec2.DescribeInstanceTypesInput{
+					InstanceTypes: []*string{aws.String("t2.micro")},
+				})
+				if err != nil {
+					return "", err
+				}
+				return *availableTypes.InstanceTypes[0].InstanceType, nil
+			default:
+				return "", err
+			}
+		}
+		return "", err
+	}
+	return *availableTypes.InstanceTypes[0].InstanceType, nil
+}
+
+func RetrieveAmi(awsClient awsclient.Client, amiOwner string) (string, error) {
+	var ami string
+	executableBy := "all"
+	input := ec2.DescribeImagesInput{
+		ExecutableUsers: []*string{&executableBy},
```

Comment:
very minor but we could probably drop the above `executableBy` variable and use `aws.String(all)` here.

Or - I also don't think it would be a bad idea to make it a constant somewhere.